docs/getting-started-guides/aws/cloud-configs/master.yaml

```diff
@@ -1,15 +1,18 @@
 #cloud-config
 
 ---
-write_files:
-- path: /opt/bin/waiter.sh
-  owner: root
-  content: |
-    #! /usr/bin/bash
-    until curl http://127.0.0.1:4001/v2/machines; do sleep 2; done
+hostname: master
 coreos:
+  etcd2:
+    name: ${DEFAULT_IPV4}
```

Comment:
This will not work. You need to depend on the [setup-network-environment](https://github.com/kelseyhightower/setup-network-environment) tool. That's why I've always used real unit files vs the cloud-init short cuts.